bios/rimantas-mocevicius.md

```diff
@@ -0,0 +1,10 @@
+# Rimantas Mocevicius
+
+![rimas](../images/headshots/rimantas-mocevicius.jpeg)
+
+[<img src="../images/icons/twitter.png" width="56px;"/>](https://twitter.com/Rimusz) [<img src="../images/icons/github.png" width="56px;"/>](https://github.com/rimusz) [<img src="../images/icons/linkedin.png" width="56px;"/>](https://linkedin.com/in/rimantasmocevicius)
+
+Rimas is big Open Source supporter. He co-founded Helm and wrote CoreOS Essentials book.
```

Comment:
I'd suggest rephrasing this a bit to: "_Rimas is a big supporter of Open Source software. He co-founded [Helm](https://helm.sh/) and wrote the CoreOS Essentials book._"

Reply:
done